Qwerty's slave work PR

Toxici11i opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->
Changes stunbatons to be more friendly, helps remove the instant stun meta of destroying liberals with one tap
## Why It's Good For The Game

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->
this was a bullshit meta that shouldve been fixed by /tg/
## Changelog
:cl:
tweak: Removed the stunbaton and stunprods "stun", instead just replaced the call with adjuststaminaloss. makes them do 90 stamina loss/70 stamina loss respectively. Also changed disablers to have 5 more damage and cost 10 less energy.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Commit:
Changes stunbatons to be more friendly to the game

makes them do 120 stamina damage on sec batons, 100 on improvised.

ghost commented:
https://youtu.be/L4v0d-rYsv8

ghost commented:
also, stun baton stun should be 140 stamina damage.

qwertyquerty commented:
How many hits will it take to knock you over at full stam

qwertyquerty commented:
Also I think we should compensate for this by buffing something else. My first idea was giving disablers more charge

Toxici11i commented:
It takes 2 hits, also this is the first "test" PR that I think I shall expand upon. The first hit puts you into "red" health - severely slowing you and could make you fall over by itself, the second WILL put you into stamina crit long enough to cuff you, and drag you quite far.
FECK I DIDNT MEAN TO CLOSE

Toxici11i commented:
Also, to compensate for the lack of a now viable 1tap wonderweapon for security used to destroy criminals, I think we should simply buff the stamina damage and charge of disablers, 5 stamina damage and an extra 2 shots should be more than enough to compensate

qwertyquerty commented:
I don't see what's changed honestly. One click slows you down enough for them to immediately and easily get a second. Then kill. I think it should take 3. And buff disablers a bunch, because disablers are amazin

Toxici11i commented:
Qwerteh, since its not just calling the paralyse proc and instead relies on stamina damage, chems and other factors can affect it, also I can just adjust the values for lets say 70 on a baton (should take 3 hits to stamina crit). Ill make a seperate PR for disabler buffs

Commit:
Changes

"Stamina damage will start to rapidly regenerate 10 seconds after last time you took such damage, and some stun-reducing chemicals can increase the regeneration speed. Your stamina-critical state (paralyze) will end when you regenerate to under 100 stamina damage." - TG
also tg (I think this is wrong) However, if you gain 100 stamina damage, you'll get inflicted with a special variety of paralyze for some time, a stamcrit version 

meaning that with the new changed version its not going to be fecken gay and 2tap

Commit:
Reduced disabler cost in ammunition

Commit:
buff disabler damage by 5

qwertyquerty commented:
I'd rather those disabler buffs go into this PR, and I think doing 90 stam damage would be fine.

Commit:
buffed again slightly

Commit:
update HOS disabler cost

Commit:
Merge pull request #1 from Toxici11i/Disabler-buffs

Disabler buffs

qwertyquerty commented:
One of my ideas was that heads of staff should have disablers to protect themselves instead of TSBs

Toxici11i commented:
ok :b:erty

Commit:
I hate you qwerty

Commit:
I hate you qwerty

Commit:
Did i say i hated you yet qwerty?

Commit:
Qwerty commit hated

Commit:
hated

Commit:
bruh

qwertyquerty commented:
I bet a bunch of sec mains are gonna downvote this because they believe sec should be perfect and there should be no mistakes. This is a good update which requires actually robust players to play sec.

ike709 commented:
> requires actually robust players to play sec.

but those players are usually smart enough to not play sec

qwertyquerty commented:
Heavily based PR

thatguythere03 commented:
>It's now harder to take down a drugged up perp
Actually accurate and introduces a whole lot more possibilities in the exchange between Sec and criminals. Good change.

qwertyquerty commented:
You should probably apply this same nerf to classic batons and telescopic batons before I merge this

Toxici11i commented:
telebatons no longer spawn on heads - and therefor dont need to be nerfed at this moment, ill just look into dets baton

qwertyquerty commented:
They should still be nerfed the same since there are ways to get them. Add those two, and I'll merge

Commit:
changes classic baton

Toxici11i commented:
It seems like the telescopic and classic batons use the same function, should be ready now. (Telebatons are just a subtype of classic batons, and i cant find any seperate attack code for it)

qwertyquerty commented:
What could possibly go wrong

ghost commented:
this is cancer, please revert this @qwertyquerty
